src/map/clif.c

```diff
@@ -25993,12 +25993,12 @@ static void clif_format_itemlink(StringBuf *buf, const struct item *it)
 #if PACKETVER >= 20161116
 	static const char view_sep = '&';
 #endif
-#if PACKETVER_MAIN_NUM >= 20200916 || PACKETVER_RE_NUM >= 20200723
+#if PACKETVER_MAIN_NUM >= 20200916 || PACKETVER_RE_NUM >= 20200723 || PACKETVER_ZERO_NUM >= 20221024
```

Comment:
@4144 / @Asheraf do you know if I've got the zero packetver right? I am assuming this date because `grade` fields were added to packets at this date. But I could not test

Reply:
for packets with grade field should be correct date. for item link idk need test in client

Reply:
ok. thanks! I will leave like that for now as an "educated guess" and try to test later. Should work for most cases.